Grammar fixes (#1966)
hacks-guide/Guide_3DS: _pages/en_US/seedminer.txt

  + This is usually fast (1-5 minutes) but in some cases, can take up to half an hour
  + During this process, the bot you added may not show up on your 3DS. As long as the website updates, this is not an issue
  + If you are still waiting after half an hour, join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp) and ask (in English) for someone there to assist you

> 1. When the process has been completed, download your `movable.sed` file from the site

Reviewer: ```suggestion
1. When the process is complete, download your `movable.sed` file from the site
```

Reviewer: This was not added.
